pkg/ddl/ddl_api.go

```diff
@@ -2977,7 +3039,7 @@ func (d *ddl) BatchCreateTableWithInfo(ctx sessionctx.Context,
 			}
 		}
 
-		job, err := d.createTableWithInfoJob(ctx, dbName, info, c.OnExist, true)
+		job, err := d.createTableWithInfoJob(ctx, dbName, info, nil, c.OnExist, true)
```

Comment:
for batch create tables we should append their involved schema to the final job

Reply:
I have confirmed with BR dev that BatchCreateTable is only used in BR, and BR makes sure no concurrent DDL API will be called. Apart from that, BatchCreateTable has no SQL statement and should not happen in TiDB DDL use case. I will add a TODO for it.

Reply:
Oh I understand you mean add involved schema `db.*`